sympy/interactive/session.py

```diff
@@ -71,7 +71,7 @@ def _make_message(ipython=True, quiet=False, source=None):
                 _source += '>>> ' + line + '\n'
 
         doc_version = sympy_version
-        if doc_version.find('-git') >= 0:
+        if doc_version.find('-git') >= 0 or doc_version.find('dev') >= 0:
```

Comment:
I guess not, but I just left it there in case. I didn't really read what all that code does. I'll look at it.